cmd/beacon/beacon.go

```diff
@@ -18,6 +18,7 @@ package beacon
 import (
 	"os"
 
+	"github.com/marlinprotocol/ctl2/cmd/keys/keystore"
```

Comment:
Program logic does not reside in `ctl2/cmd/`. This location is reserved only for "CLI creation" and not business logic. Use  `ctl2/modules/` for hosting such.